resources/kcp/charts/kyma-metrics-collector/templates/public-cloud-info-config-map.yaml

```diff
@@ -199,6 +199,42 @@ data:
             "memory": 256,
             "storage": 1600,
             "max_nics": 8
+        },
```

Comment:
The following are missing:
- Standard_D2s_v5
- Standard_D96s_v5

Reply:
They are not supported. "should modified to offer 4-64 vCPU machine types"